contracts/permissionless/OrdersInterface.sol

```diff
@@ -4,14 +4,15 @@ pragma solidity 0.4.18;
 interface OrdersInterface {
     function getOrderDetails(uint32 orderId) public view returns (address, uint128, uint128, uint32, uint32);
     function add(address maker, uint32 orderId, uint128 srcAmount, uint128 dstAmount) public returns (bool);
-    function removeById(uint32 orderId) public returns (bool);
+    function remove(uint32 orderId) public returns (bool);
     function update(uint32 orderId, uint128 srcAmount, uint128 dstAmount) public returns (bool);
     function getFirstOrder() public view returns(uint32 orderId, bool isEmpty);
     function allocateIds(uint32 howMany) public returns(uint32);
     function getTailId() public view returns(uint32);
     function getHeadId() public view returns(uint32);
     function findPrevOrderId(uint128 srcAmount, uint128 dstAmount) public view returns(uint32);
 
+
```

Comment:
why this extra space?
@amitkot

Reply:
Added by mistake during merge, removed them now. Thanks!